optuna/visualization/_parallel_coordinate.py

```diff
@@ -97,6 +98,7 @@ def _get_parallel_coordinate_plot(
 ) -> "go.Figure":
 
     layout = go.Layout(title="Parallel Coordinate Plot")
+    reversed_scale = _is_reverse_scale(target, study.direction)
```

Comment:
If `target` is `None`,  `target` is overwritten at L122, so we need to store `reversed_scale` here.